Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -644,10 +649,12 @@ private void OnKeyDown(object sender, KeyEventArgs e)
             switch (e.Key)
             {
                 case Key.Down:
+                    isArrowKeyPressed = true;
```

Comment:
Just did a quick test and found that holding Next Result hotkey also has this issue. IMO we should just left it. hard to fix for every hotkey.

Reply:
I agree with you. Basically, the user mainly uses the single key, but the shortcut key that combines the two will not be used. (Pressing key situation) I think we can ignore other than this key.